javascript/javascript1/week3/index.js

```diff
@@ -0,0 +1,273 @@
+//FREECODECAMP EXERCISES
+//profile name - natsgt
+
+//ITEM ARRAY REMOVAL
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+
+// Write some code here
+function removeName(name) {
+  for (let i = 0; i < names.length; i++) {
+    if (names[i] === name) {
+      names.splice(i, 1);
+    }
+  }
```

Comment:
Javascript can search for what you want without using a for loop. Always assume someone has done the work for you. 👍
```suggestion
const nameToRemove = "Ahmad";
const index = names.indexOf(nameToRemove);
names.splice(index, 1);
```

Reply:
Thanks for the tip.